### PR TITLE
impl command R_BETWEEN

### PR DESCRIPTION
**Pull request by kriw**
Related to issue #7892 
This command `?btw` takes 3 arguments a, b, c and returns 1 if a < b && b < c else returns 0.
If the command is failed, it prints its usage, and returns undefined value.
Each argument is required to be either number or "(" expr ")".
```
[0x00000000]> ?btw 1 2
Usage: ?btw num|(expr) num|(expr) num|(expr)
[0x00000000]> ?btw 1 2 3
[0x00000000]> ??
1
[0x00000000]> ?btw 1 2 (1 + 2)
[0x00000000]> ??
1
[0x00000000]> ?btw 3 2 1
[0x00000000]> ??
0
```

### CODE REVIEW

**Comment by Maijin** [comment]
@kriw https://travis-ci.org/radare/radare2/jobs/298769198#L677

**Comment by Maijin** [comment]
For identation use ./sys/ident.sh read CONTRIBUTING.md

File: libr/util/unum.c [hunk 1]
```diff
@@ -580,3 +580,29 @@ R_API ut64 r_num_tail(RNum *num, ut64 addr, const char *hex) {
...
+			}
+			if (*input_value == '(') {
+				num_parens++;
+			} else if (*input_value == ')') {
+				num_parens--;
+			}
+		}
+		ns[i] = r_num_math (num, input_value);
+	}
+	return num->value = ns[0] < ns[1] && ns[1] < ns[2]
```
Comment (radare) [inline libr/util/unum.c]: There’s R_BETWEEN macro that does that

**Comment by radare** [comment]
Change this line to use the rbetween and ill
Merge. Good patch :)

**Comment by radare** [comment]
Can you push some tests for the r2r repo? Thanks!

**Comment by kriw** [comment]
OK, I'll implement it.

**Comment by kriw** [comment]
@radare R_BETWEEN macro return the result of `x <= y && y <= z`, but it evaluate x < y && y < z right ?

**Pull request merged**